src/libraries/System.Net.Security/src/Resources/Strings.resx

```diff
@@ -440,6 +440,9 @@
   <data name="net_ssl_app_protocol_invalid" xml:space="preserve">
     <value>The application protocol value is invalid.</value>
   </data>
+  <data name="net_ssl_app_protocol_config_notsupported" xml:space="preserve">
```

Comment:
This surprises me, since there are already platforms where ALPN isn't supported.  Is there really not already a string to use?

Reply:
Ah, I got the expected return value mixed up when I was looking at the Unix implementation. The other platforms silently ignore `ApplicationProtocols` if they don't support setting it. I'll make Android match that.